docs/edex/distributed-computing.md

```diff
@@ -119,22 +123,22 @@ Since this Database/Request server is not running the main *edexIngest* JVM, we
 
 ---
 
-### Ingest/Decode Server
+### Ingest/Decode Server (Ancillary EDEX Server)
 
 For this example, this server will be referred to by the IP address **10.0.0.7**.
+The **Main EDEX** server will be referred to by the IP address **10.0.0.9**.
 
 #### 1. Install
 
-	groupadd fxalpha && useradd -G fxalpha awips
-	wget -O /etc/yum.repos.d/awips2.repo https://downloads.unidata.ucar.edu/awips2/current/linux/awips2.repo
-	yum clean all
-	yum groupinstall awips2-ingest
+	wget https://downloads.unidata.ucar.edu/awips2/current/linux/awips_install.sh
+	chmod 755 awips_install.sh
+	sudo ./awips_install.sh --ingest
 
 #### 2. EDEX Config
 
 `vi /awips2/edex/bin/setup.env`
 
-Here you should redefine `DB_ADDR` and `PYPIES_SERVER` to point to the **Database/Request** server (10.0.0.9)
+Here you should redefine `DB_ADDR` and `PYPIES_SERVER` to point to the **Main** or  **Database/Request** server (10.0.0.9) and the `EDEX_SERVER` to point to the current **Ingest** server (10.0.0.7)
```

Comment:
I'm confused about this statement...?  Where is this `EDEX_SERVER` variable?  We don't have this on either of our edex2 or radar2 instances?

Reply:
that's what was already in the documentation. I updated it to EXT_ADDR

Reply:
I see that in the line below, but why do you still mention EDEX_SERVER in the wording above the code excerpt?